ui/pom.xml

```
@@ -39,7 +39,8 @@
     </xwiki.extension.licensing.publicDocuments>
     <xwiki.extension.licensing.excludedDocuments>
       IdentityOAuth.Config,
-      IdentityOAuth.WebPreferences
+      IdentityOAuth.WebPreferences,
+      IdentityOAuth.IdentityOAuthGeneralConfiguration
```

I'm not sure that "General" bring much. Why not keep it simple:

```suggestion
      IdentityOAuth.IdentityOAuthConfiguration
```

Probably because we have ``IdentityOAuthConfigClass`` which should have been named ``IdentityOAuthProviderConfigClass``...

---

So it won't be too confusing, given the other `IdentityOAuthConfig` pages?

---

Actually, I will need to add a `IdentityOAuthGeneralConfigClass` for #19 , so I'm not sure I can drop the `General`. I will be to confusing given the already existing `IdentityOAuthConfigClass`

---

OK, let's keep it like this then.